platform/android/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ Mapbox welcomes participation and contributions from everyone.  If you'd like to
 
 ## master
 
+## 7.0.1 - January 11, 2019
+ - Add setMaximumFps configuration on MapView [#13498](https://github.com/mapbox/mapbox-gl-native/pull/13498)
```

Comment:
Could we make a note that this is an early access API? The actual functionality will be officially released with 7.1.0.